functional_tests/tests.py

```diff
@@ -31,9 +35,10 @@
 class FunctionalTest(StaticLiveServerTestCase):
     @classmethod
     def setUpClass(cls):
+        browser_path = "./node_modules/chromedriver/lib/chromedriver/chromedriver"
```

Comment:
And it's probably `DRIVER_PATH` not `BROWSER_PATH` - anyway i like that you used descriptive variable instead of hardcoding it :)